### PR TITLE
OneLogin SSO documentation updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added updated documentation for OneLogin SSO

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [OneLogin documentation updates](https://www.notion.so/rudderstacks/OneLogin-Documentation-Update-4daac0ce2495427189e62c5d04629227)

### CODE REVIEW

File: docs/user-guides/administrators-guide/onelogin-sso.mdx [hunk 1]
```diff
@@ -2,91 +2,108 @@
...
 
-<!--![](../../.gitbook/assets/image (58).png)-->
+The settings to be configured are listed in the following table:
 
-<img src="../../assets/image (58).png" alt="Search and select RudderStack" />
+| Setting | Value |
+| :-------| :------|
+| Audience (EntityID) | `urn:amazon:cognito:sp:us-east-1_ABZiTjXia` |
+| Recipient | `https://auth2.rudderstack.com/saml2/idpresponse` |
+| ACS (Consumer) URL Validator | `^https:VVauth2\.rudderstack\.comVsamI2VidpresponseV\S` |
```
Comment (aris1009) [inline docs/user-guides/administrators-guide/onelogin-sso.mdx:48]:
```suggestion
| ACS (Consumer) URL Validator | `^https:\/\/auth2\.rudderstack\.com\/saml2\/idpresponse\/\$` |
```

Comment (aris1009) [reply docs/user-guides/administrators-guide/onelogin-sso.mdx:48]: This is a regex pattern and some characters need to be escaped

**Review by aris1009 — CHANGES_REQUESTED** [review]
Looks awesome! Just a value needs to be updated

**Comment by ameypv-rudder** [comment]
> Looks awesome! Just a value needs to be updated

thanks Aris - I've pushed the suggestion - LMK if we're good to go here.

**Review by aris1009 — APPROVED** [review]
lgtm

File: docs/user-guides/administrators-guide/onelogin-sso.mdx [hunk 2]
```diff
@@ -2,91 +2,108 @@
...
-  This **SSO** section contains information which you will be required to share
-  with us so that we can enable the SSO for your organization.
+For the <strong>LastName</strong> custom attribute, you can specify a single field <code class="inline-code">Name</code> - which specifies how you would like to see your employees on the RudderStack web app.
 </div>
 
-- Retrieve the contents of the following two data fields:
-  - **Issuer URL**
-  - **SAML 2.0 Endpoint \(HTTP\)**
-- Now, look for the field **X.509 Certificate** and click on **View Details** underneath, as shown:
+9. To add a custom parameter, simply click the **+** button, enter the **Field name**, and select the value from the dropdown, as shown:
```
Comment (ishachopra0401) [inline docs/user-guides/administrators-guide/onelogin-sso.mdx:83]:
```suggestion
9. To add any other custom parameter, click the **+** button, enter the **Field name**, and select the value from the dropdown, as shown:
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**